.drone.yml

```diff
@@ -9,7 +9,9 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "s3-to-redshift"
+    applications: 
+    - "s3-to-redshift"
+    - "s3-to-redshift-fast-prod"
```

Comment:
nit: I'd name it just `fast` .. everything should support running in both dev and prod